scripts/custom_targeting_to_coverage.R

```diff
@@ -0,0 +1,74 @@
+##this script creates a coverage table (proportion of target population vaccinated at admin 0 level) based on information inside 
```

Comment:
Have you tested the script? After reviewing the comments and making edits, could you please send an "input" targeting table and "output" coverage table to the channel (created by running the script) so we can verify it works correctly? You can use any small dummy input targeting table, perhaps by taking a few rows from the expected DRC realistic OCV scenario